src/schema/v2/artwork/index.ts

```diff
@@ -731,6 +731,17 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           )
         },
       },
+      pricePaid: {
+        type: Money,
+        description:
+          "The price paid for the artwork in a user's 'my collection'",
+        resolve: (artwork) => {
+          return {
+            cents: artwork.price_paid_cents,
+            currency: artwork.price_paid_currency || "USD",
```

Comment:
I'm not sure the `||` part is needed here. If we fix the `display` problem, it may not be needed.